### PR TITLE
VACMS-9387: update content Get Medical Records page - Aug 18 target merge

### PR DESCRIPTION
**Pull request by allisonlu**
**TARGET MERGE**: August 18th, for deploy on the 19th.

## Description
Updated FAQ copy on /health-care/get-medical-records/ with new content to reflect new Research & Support pages:

- [Get.your.medical.records_Cerner.version_for PW (1).docx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/8970419/Get.your.medical.records_Cerner.version_for.PW.1.docx)
- [Get.your.medical.records_Non-Cerner.version_for.SME.team.docx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/8970472/Get.your.medical.records_Non-Cerner.version_for.SME.team.docx)

## Original issue(s)
[/department-of-veterans-affairs/va.gov-cms/issues/9387](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9387)

## Testing done
- [x] locally
- [ ] Get QA reviews from @RLHecht and @laurwill from Sitewide Content
- [ ] @jilladams to get test user for Cerner/authenticated content

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by jhonny218 — APPROVED** [review]

**Comment by RLHecht** [comment]
@allisonlu I left comments in our slack thread about some content changes that need to happen. A few links, etc. Also, all of the Veterans Health Information Exchange content that's on the original pages should be removed as we now have a new page where we moved this content. Just confirming that's the case.

**Comment by allisonlu** [comment]
@RLHecht @laurwill 

Made the updates listed in Slack:
[x] removed "Please" from Sign In CTA component
[x] updated /resources/the-veterans-health-information-exchange-vhie links

**Comment by allisonlu** [comment]
@RLHecht 

I made the above changes except for the first bullet for Cerner. I made a note yesterday on the issue: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9387#issuecomment-1169331716

> I've left the [dynamic facility name] alone, since this would take more time than expected to figure out how to pull in the current facility name.

I added the em dash though

**Comment by RLHecht** [comment]
@allisonlu Now that we didn't publish last week I was curious if you were able to figure out how to handle the dynamic facility name content.

**Comment by laurwill** [comment]
Hi @allisonlu ! We have finally gotten approval to publish these pages, with just 1 link change we need to make.

In the accordion "What if I can't access all of my medical records through My VA Blue Button?" (or "in My Healthevet or My VA Health" in Cerner), we need to change the second link as follows:
- **From** "Learn how to get medical records from DoD"
- **To** "Learn how to get DoD Health Records on the TRICARE website"

And the new URL for that link is www.tricare.mil/resources/medicalrecords

Once you've made that change, could you post a new version here for our final copyedit?

We're aiming to have these go live by next Friday, 8/19, but let us know if that's not doable. And thanks so much for all your work on this!

**Comment by laurwill** [comment]
Hi @allisonlu and @ryguyk ,

Just documenting a content update from our slack thread. It turns out the line that mentions Mann Grandstaff is hard coded, not dynamic. Please change that line:
- From: "If you receive care at both Mann-Grandstaff and another VA facility, you may need to use both portals."
- To: "If you receive care at more than one VA facility, you may need to use both portals."

I believe that's the only instance of the hard-coded "Mann-Grandstaff" facility name on the page, so that should be the only update needed to fix this problem. Thank you!

**Comment by allisonlu** [comment]
@RLHecht @laurwill 

Made the following updates:
- [x] updated copy and link for DoD accordion (for both pages)
- [x] updated hard-coded instance of "Mann-Grandstaff"

## Non-Cerner version
Full page screenshot of Get Medical Records page for non-Cerner users


## Cerner version
Full page screenshot of Get Medical Records page for Cerner users

**Comment by laurwill** [comment]
Hi @allisonlu ! Thanks for providing these updated screenshots.

It looks like both versions are now showing the Cerner content. The version labeled "Non-Cerner" above has all Cerner content (mentioning both My VA Health and My HealtheVet), and the version labeled "Cerner" has mostly Cerner content but with the Non-Cerner H2 "Use VA Blue Button to manage your records online." 

Could you update to make sure each version has the correct Cerner/Non-Cerner content and post new screenshots for review? (Sorry this content is such a pain to keep straight!)

Also, we had made some small text updates to the sign-in widgets, which aren't visible in these screenshots — what would be the best way to review those updates? 

Thanks for all your work on this!

**Comment by allisonlu** [comment]
Hi @laurwill! Thanks for being patient as my local environment was rebuilding. I realized I messed up the screenshots in the previous comment, but if you look at the **Files changed**, the updates are correct. 

Here are the correct screenshots:

## Non-cerner
non-cerner_not-logged-in_get-medical-records

## Cerner
cerner_not-logged-in_get-medical-records_

**Comment by micahchiang** [comment]
Was `main` properly merged into this branch? Seems like there are a lot of changes in this PR unrelated to faq copy updates on  /health-care/get-medical-records/

File: babel.config.json [hunk 1]
```diff
@@ -92,20 +92,10 @@
 
           "@department-of-veterans-affairs/platform-landing-pages/exports" : "./src/platform/landing-pages/exportsFile.js",
           "@department-of-veterans-affairs/platform-landing-pages/dev-template" : "./src/platform/landing-pages/dev-template.js",
-          "@department-of-veterans-affairs/platform-landing-pages/facility-sidebar" : "./src/platform/landing-pages/facility-sidebar.json",
-          "@department-of-veterans-affairs/platform-landing-pages/header-footer-data" : "./src/platform/landing-pages/header-footer-data.json",
+          "@department-of-veterans-affairs/platform-landing-pages/facility-sidebar" : "./src/platform/landing-pages/facility-sidebar.jason",
+          "@department-of-veterans-affairs/platform-landing-pages/header-footer-data" : "./src/platform/landing-pages/header-footer-data.jason"
```
Comment (Mottie) [inline babel.config.json:96]: This isn't right, is it?

**Comment by allisonlu** [comment]
@micahchiang nope, `main` was not merged in correctly 😅. i'm going to scrap this PR and open a new PR. sorry about the notifications!

File: babel.config.json (hunk 1 above)
Comment (allisonlu) [reply babel.config.json:96]: nope, i'm going to scrap this PR and open a new PR. sorry about the notifications!

**Comment by allisonlu** [comment]
i'm going to scrap this PR and open a new PR. @all sorry about the notifications!

**Pull request closed without merging**